src/main/java/de/tum/in/www1/artemis/config/SecurityConfiguration.java

```diff
@@ -164,12 +165,27 @@ protected void configure(HttpSecurity http) throws Exception {
         .and()
             .authorizeRequests()
             .antMatchers("/api/register").permitAll()
+            .antMatchers("/api/{v\\d*}/register").permitAll()
```

Comment:
The other regex in `VersionRangesRequestCondition` got changed to `\d+`. Should the patterns in this file also be changed for consistency?
```suggestion
            .antMatchers("/api/{v\\d+}/register").permitAll()
```